pawan/html/practice_assignment/Programming_Exercises4.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>Resume</title>
+        <style> 
+            table,th,td{
+            border: 2px solid black;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>Pawan Sharad Jagtap</h1>
+        <p><strong>pavanj10may@gmail.com</strong></p>
+        <hr>
+        <h3>*About Me*</h3>
+        <p>My name is <i>Pawan Sharad jagtap</i>.I'm 23 years old. I'm from Pune.<br> currently pursuing my degree in <em>BCS</em> from swaraj college of science and computer studies. </p>
+        <h3>Work</h3>
+        <p>
+        <b><em>I'm Fresher</em></b> 
+        </p>
+        <h3>To Do List </h3>
+        <a href="todolist.html" target="_blank">Click here </a>
+        <h3>Stuff I would like to do someday</h3>
+        <a href="stuffIwouldliketodosomeday.html" target="_blank">CLick here </a>
+        <h3>Interest</h3>
+            <ul style="list-style-type:square;">
+                <li>swimming</li>
+                <a href="https://en.wikipedia.org/wiki/Swimming" target="_blank">
```

Comment:
wrong practice. Did you really understand the child-parent relation?

Reply:
saurabh please give them the suggestions instead of asking questions